src/pages/home/report/ReportFooter.js

```diff
@@ -102,5 +102,6 @@ export default compose(
     withWindowDimensions,
     withOnyx({
         shouldShowComposeInput: {key: ONYXKEYS.SHOULD_SHOW_COMPOSE_INPUT},
+        initialValue: false,
```

Comment:
In this case we believed not, otherwise the composer would be shown on every single page that gets opened. Since this was blocked to render since it was waiting for onyx, the default value would not matter, but now it does, wdyt?